api/src/api/users.ts

```diff
@@ -187,14 +187,63 @@ router.put(
   }),
 );
 
+// Get all pending users
+router.get(
```

Comment:
this will create a route that looks like 

`https://south-side-weekly.vercel.app/api/users/users/pending`

I suggest:

`https://south-side-weekly.vercel.app/api/users/all/pending`